agent/rpm/patches/stockpile-shebang.patch

```diff
@@ -0,0 +1,16 @@
+--- a/agent/stockpile/roles/openshift-cluster-topology/files/openshift_config_scraper.py	2019-07-03 15:36:28.855974709 -0400
```

Comment:
This patch has always bugged me; is this a permanent workaround, or a temporary workaround?

Reply:
Temprorary - but as my father used to say: "There's nothing more permanent than a temporary fix..."